### DISCUSSION TITLE
a problem with not being able to specify nickname for published servers

### DISCUSSION THREAD

**Discussion by peter1384** [BearWare/TeamTalk5#2276] [Ideas]
@CoBC when we press the edit action for a published server, we get the case view server information type in the serverdlg, and even the nickname box which is optional in there is read only, so even if that server allowes, we can't specify a nickname in there. should i modify that part so in the view server information dialog, only the nickname edit box is not readonly so we can wright on it, also add the save button, or it's not a good idea? i don't think it's a good idea because the information for the published servers are not being saved in our teamtalk settings.
what do you think?

**Comment by CoBC** [comment]
I think it's better to keep it like this, we can already specify 
nickname by pressing F5 or go to "Me" menu and "Change nickname" when 
connected to server

**Comment by peter1384** [comment]
@CoBC ok. I want to say another thing. Is it not better to use tree view instead of combo box for the move users dialog? that way channels can be collapsed and expanded just like normal and easya, raather than all channels and their subchannels listed in a long combo box

**Reply by bear101** [reply]
you can typo on your keyboard to get the item you want in a combobox

**Reply by bear101** [reply]
it's faster than navigating
